doc/en/user/source/extensions/vectortiles/install.rst

```diff
@@ -7,7 +7,7 @@ Installing the Vector Tiles Extension
 
    .. warning:: Make sure to match the version of the extension to the version of GeoServer.
 
-#. Extract the archive and copy the contents into the GeoServer :file:`WEB-INF/lib` directory.
+#. Extract the archive and copy the contents into the GeoServer :file:`webapps/geoserver/WEB-INF/lib` directory.
```

Comment:
The "webapps" directory is commonly found in Tomcat/Jetty, and in our "bin" package, but it's not guaranteed to be found in other web containers, e.g., Wildfly and WebSphere do not have it.

Rewording the instruction might be useful, but it should be done by adding an extra sentence, something like:

> For example, if you are using the GeoServer bin package, or the Windows installer, look for `webapps/geoserver/WEB-INF/lib`

Reply:
Afterwards I found that the information is contained inside the README.html of the plugin archieve. If this is consistent for all platforms, it could just reference that?

Reply:
The READE has correctly contextualized information:

> Unzip this archive into the GeoServer library directory. In the binary
   install this is under ``<GEOSERVER_ROOT>/webapps/geoserver/WEB-INF/lib``.

The first part is generic (even more so than the existing doc for vector tiles), the second one refers specifically to the binary (bin) package, where we know where GeoServer ends up being.  It's so small that I'd rather copy it instead of having people look first at the docs and then also the zip.